apps/fulfillment/mock/db.json

```diff
@@ -979,5 +979,30 @@
         }
       }
     ]
+  },
+
+  "push-notification-subscriptions": {
+    "data": {
+      "type": "push-notification-subscriptions",
+      "id": "id",
+      "attributes": {
+        "providerName": "web-push-php",
+        "payload": {
+          "endpoint": "",
+          "expirationTime": null,
+          "keys": {
+            "p256dh": "p256dh",
+            "auth": "auth"
+          }
+        },
+        "group": {
+          "name": "warehouse",
+          "identifier": "identifier"
+        }
```

Comment:
This is a payload that FE sends, please replace this with a response that BAPI sends back.

Reply:
No, it is the response that BAPI sends back.
<img width="804" alt="image" src="https://github.com/spryker/oryx/assets/43376934/8f9b8bbc-4bf5-4dba-ad72-7f51c4a08253">